app/helpers/audits_helper.rb

```diff
@@ -257,9 +257,32 @@ def key_to_association_class(key, auditable_class)
     reflection_obj ? reflection_obj&.klass : nil
   end
 
+  def find_owner_class(key, change, audit)
+    type = nil
+    if audit.audited_changes.has_key?('owner_type')
```

Comment:
While I was testing I've never went to this branch for some reason...

Reply:
Have you switched the owner from a user to a user group or the other way around?